types/derivatives.graphql

```diff
@@ -63,10 +63,10 @@ type DerivativeExchangeTicker {
     funding_rate: BigNumber!
     open_interest_usd: BigNumber!
     h24_volume: BigNumber!
-    converted_volume: ConvertedVolume!
-    converted_last: ConvertedLast!
-    last_traded: Int!
-    expired_at: Int
+    converted_volume: Map! @annotate(type: "Map<String!, BigNumber>!")
```

Comment:
The `ConvertedVolume` and `ConvertedLast` types were fairly small. Do they still cause problems that are fixed by using `Map`? It might be nicer to keep them as types with known properties if there are no issues.

Reply:
Having a generic map is better because if in the future or for other coins if there are more keys than we typed then it will fail so that's why I kept it map.